### ISSUE TITLE
Consider adding crossOrigin = "anonymous" to Image.fromURL to prevent tainted canvases

### ISSUE THREAD

**Issue by benjaminjackman** [konvajs/konva#382]
Hi, It might be advisable to enable crossOrigin loading to the fromURL function in Konva.Image

https://github.com/konvajs/konva/blob/6d909f833f69f909c1bb3afab714942650eed709/src/shapes/Image.js#L234-L243

This brief article lays out why, but long story short having an additional line under
https://blog.codepen.io/2013/10/08/cross-domain-images-tainted-canvas/

```javascript
img.src = url;
``` 
of 
```javascript
img.crossOrigin="Anonymous";
```
could help prevent the canvas from getting tainted

( one more article https://developer.mozilla.org/en-US/docs/Web/HTML/CORS_enabled_image )

**Comment by humblecoder** [comment]
For some reason I'm still receiving the CORS error.  I also tried adjusting `crossOrigin` to `null` after the fact, although it's probably too late within the callback.  Please advise.

**Comment by lavrton** [comment]
`crossOrigin` do not prevent all CORS error. You may have to fix it from the server side.

**Comment by humblecoder** [comment]
@lavrton apologies, you're correct.  Although I had CORS middleware for Laravel, it was a public resource, so it was only being touched by Nginx.  I had to add an exception in Nginx.  Thanks.

**Comment by hiratariq** [comment]
I was working on a project the other day when I kept getting CORS error while using Konva.Image.fromURL but when went back to image.onload method it working. For some reason having crossOrigin set to '', null or 'anonymous' was the issue. I needed to set nothing to get it to work. So I think if there is a higher level Konva.Image.crossOrigin kind of a thing that user can set once in their project depending on their needs that would be better.
